docs/react-v9/contributing/rfcs/shared/build-system/stop-styles-transforms.md

```diff
@@ -0,0 +1,162 @@
+# RFC: Stop pre-processing styles with Griffel in `@fluentui/react-components`
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31134&commitId=5d95695df9e66e3f0405a4e6a73814ac49eb0959&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.image-bad-url+icon.normal.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>